README_fr.md

```diff
@@ -0,0 +1,1641 @@
+
+```sh
+$ git log -S "chaîne de caractères à rechercher"
+```
+
+Paramètres communs :
+
+* `--source` sert à montrer le nom du ref en se basant sur la ligne de commande avec laquelle chaque commit a été atteint.
+
+* `--all` sert à commencer depuis chaque branche.
+
+* `--reverse` retourne les résultats dans l'ordre inverse, c'est à dire que la commande affichera le premier commit qui a fait la modification.
+
+<a name="i-want-to-find-by-author-committer"></a>
+### Je veux rechercher par auteur·trice/validateur·trice
+
+Pour rechercher des commits par auteur·trice/validateur·trice, vous pouvez utiliser :
+
+```sh
+$ git log --author=<nom ou email>
+$ git log --committer=<nom ou email>
+```
+
+Gardez en tête que l'auteur·trice et la personne qui a créé le commit ne sont pas les mêmes personnes. L'option `--author` correspond à la personne qui a écrit le code, et, d'un autre côté, l'option `--committer`, correspond à la personne qui a commité le code au nom de l'autreur·trice.
+
+### Je veux lister les commits ayant des fichiers spécifiques
+
+Pour trouver tous les commits contenant un fichier spécifique, vous pouvez utiliser :
+
+```sh
+$ git log -- <chemin du fichier>
+```
+
+En général, vous préciseriez un chemin exact, mais vous pouvez aussi utiliser des wildcards dans le chemin et le nom du fichier :
+
+```sh
+$ git log -- **/*.js
+```
+
+En utilisant des wildcards, il est utile de préciser `--name-status` pour voir la liste des fichiers commités :
+
+```sh
+$ git log --name-status -- **/*.js
+```
+
+### Trouver un tag où un commit est référencé
+
+Pour trouver tous les tags contenant un commit en particulier :
+
+```sh
+$ git tag --contains <idducommit>
+```
+
+## Sous-modules
+
+<a name="clone-submodules"></a>
+### Cloner tous les sous-modules
+
+```sh
+$ git clone --recursive git://github.com/foo/bar.git
+```
+
+Si ces derniers ont déjà été clonés :
+
+```sh
+$ git submodule update --init --recursive
+```
+
+<a name="delete-submodule"></a>
+### Retirer un sous-module
+
+Créer un sous-module est assez trivial, mais le supprimer l'est moins. Les commandes dont vous avez besoin sont :
+
+```sh
+$ git submodule deinit nomdusubmodule
+$ git rm nomdusubmodule
+$ git rm --cached nomdusubmodule
+$ rm -rf .git/modules/nomdusubmodule
+```
+
+## Objets divers
+
+### Récupérer un fichier supprimé
+
+Trouvez le commit le plus récent où le fichier existait encore :
+
+```sh
+$ git rev-list -n 1 HEAD -- nomdufichier
+```
+
+Puis faites un `checkout` sur ce fichier :
+
+```
+git checkout idducommitquiasupprimé^ -- nomdufichier
+```
+
+### Supprimer un tag
+
+```sh
+$ git tag -d <nom_du_tag>
+$ git push <remote> :refs/tags/<nom_du_tag>
+```
+
+<a name="recover-tag"></a>
+### Récupérer un tag supprimé
+
+Si vous voulez récupérer un tag qui a déjà été supprimé, vous pouvez le faire en suivant ces étapes : Tout d'abord, vous devez retrouver le tag inaccessible en question :
```

Comment:
```suggestion
Si vous voulez récupérer un tag qui a déjà été supprimé, vous pouvez le faire en suivant ces étapes :  
Tout d'abord, vous devez retrouver le tag inaccessible en question :
```
The colon `:` does not introduce a new sentence, therefore there's not need to capitalize the following word. But in this case, it feels rather bad, 'cause the next phrase also ends with another `:`... I think we'd be better off just spreading them out onto two lines.